documentation/src/docs/basic-reader-and-writer.md

```diff
@@ -9,29 +9,24 @@ You may obtain a copy of the License at
 -->
 # Working with Pravega: Basic Reader and Writer
 
-Lets examine how to build simple Pravega applications.  The simplest kind of
-Pravega application uses a Pravega Reader to read from a Pravega Stream or a
-Pravega Writer that writes to a Pravega Stream.  A simple example of both can be
-found in the Pravega Samples "hello world" app. These sample applications
-provide a very basic example of how a Java application could use the Pravega
+Lets examine how to build simple Pravega applications. The simplest kind of Pravega application uses a Pravega Reader to read from a Pravega Stream or a
+Pravega Writer that writes to a Pravega Stream. A simple sample application example of both can be
```

Comment:
`simple sample application example` -> remove `example` as is redundant.

Reply:
Accepted Suggestion